sortinghat/core/migrations/0009_auto_20210622_1033.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2 on 2021-06-22 15:33
```

Comment:
We started using Django 3.1. Please squash the commit you have later with this one.